scripts/data_flow/push_data.py

```diff
@@ -0,0 +1,68 @@
+from _setup import Setup #setup, myLogger
+import sys, os,glob
+import  _GDrive, _process,_NAS,_FTP,_zip
+sys.path.insert(0, os.environ['PROJ_DIR'] + '/' + os.environ['PROJECT'] + '/' + os.environ['AIRCRAFT'] + '/scripts')
```

Comment:
@sararunkel Once you resolve how to gracefully catch if an env var isn't set in your test suite, please add that same error catching here. Right now, if AIRCRAFT isn't set, the code crashes:

> Traceback (most recent call last):
>   File "/Users/janine/dev/aircraft_projects/scripts/data_flow/push_data.py", line 1, in <module>
>     from _setup import Setup #setup, myLogger
>     ^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/Users/janine/dev/aircraft_projects/scripts/data_flow/_setup.py", line 8, in <module>
>     sys.path.insert(0, os.environ['PROJ_DIR'] + '/' + os.environ['PROJECT'] + '/' + os.environ['AIRCRAFT'] + '/scripts')
>                                                                                     ~~~~~~~~~~^^^^^^^^^^^^
>   File "<frozen os>", line 714, in __getitem__
> KeyError: 'AIRCRAFT'

Reply:
Added in the same check here